apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -1,26 +1,83 @@
+import { usePathname, useRouter } from "next/navigation";
+import { useCallback, useState } from "react";
+
+import { useCompatSearchParams } from "@calcom/embed-core/src/useCompatSearchParams";
 import { getLayout } from "@calcom/features/settings/layouts/SettingsLayout";
 import CreateNewOutOfOfficeEntryButton from "@calcom/features/settings/outOfOffice/CreateNewOutOfOfficeEntryButton";
-import { OutOfOfficeEntriesList } from "@calcom/features/settings/outOfOffice/OutOfOfficeEntriesList";
+import {
+  OutOfOfficeEntriesList,
+  OutOfOfficeTab,
+} from "@calcom/features/settings/outOfOffice/OutOfOfficeEntriesList";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { MembershipRole } from "@calcom/prisma/enums";
 import { trpc } from "@calcom/trpc/react";
-import { Meta, SkeletonText } from "@calcom/ui";
+import { Meta, SkeletonText, ToggleGroup } from "@calcom/ui";
 
 import PageWrapper from "@components/PageWrapper";
 
 const Page = () => {
   const { t } = useLocale();
 
+  const searchParams = useCompatSearchParams();
+  const router = useRouter();
+  const pathname = usePathname();
+
+  // Get a new searchParams string by merging the current searchParams with a provided key/value pair
```

Comment:
Hey, always add a comment if it is very important or a complex thing to understand. I found some comments you added that are understandable. Pls remove

Reply:
removed